services/scripts/build_libs.sh

```diff
@@ -6,8 +6,8 @@ CLI_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 source $CLI_HOME/utils.sh
 
 LIBRARIES=(
-  "common"
   "types"
+  "common"
```

Comment:
also add types as a dependency of common in watch_libs script

Reply:
done